src/main/java/com/microsoft/aad/adal4j/AdalAuthorizatonGrant.java

```diff
@@ -62,6 +62,12 @@
         if (this.params != null) {
             outParams.putAll(this.params);
         }
+        
+        if(!outParams.containsKey("scope"))
+        {
+        	outParams.put("scope", "openid");

```

Comment:
Sounds good, but I also want to know whether you are enforcing, "We also do not allow scopes to be set by the user."